security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/function/oci-datasafe-audit-to-logging/build-fn.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+. ./set-local-vars.sh
```

Comment:
This will fail if run from a different directory.

Reply:
update the code in according with you comment